config/templates/folio_holdings.yml

```diff
@@ -0,0 +1,8 @@
+development:
+  location_codes:
+    library1: library1-folio-location-code
```

Comment:
Do the keys in `location_codes` need to be specific values? E.g., is it okay to put "Avery", or does it need to be something that can be matched to some value somewhere? If the answer is yes, can there be comments in this file?

Reply:
The keys represent ASpace repository codes (like NNC-RB) that map to a specific location in FOLIO (stored as a UUID since that's what the FOLIO API expects).

I consulted Eric about this yaml file being a bit confusing since the location codes (and other values) don't really need to be hidden. He suggested we commit it to version control and have environment-specific overrides later (if FOLIO test and prod API go out of sync for some reason). 

You can see the original yaml file with a comment explaining what key/values are used [here](https://github.com/cul/folio_sync/pull/36/files#diff-d84e9ca6f0b681a3aa3f7d969b91ea9ecb59f840912856b5655d461913cbd582). I also added FOLIO location codes on the right. Hopefully these changes make this file easier to read.

Reply:
Awesome, thank you! This is very clear.